src/control/lib/control/mocks.go

```diff
@@ -525,7 +525,8 @@ func MockFormatResp(t *testing.T, mfc MockFormatConf) *StorageFormatResp {
 
 		for j := 0; j < mfc.ScmPerHost; j++ {
 			if _, failed := mfc.ScmFailures[j]; failed {
-				if err := hem.Add(hostName, errors.Errorf("/mnt/%d format failed", j+1)); err != nil {
+				err := hem.Add(hostName, errors.Errorf("/mnt/%d format failed", j+1))
```

Comment:
nit: This kind of change is unnecessary and adds to reviewer burden, because we have to study the change to figure out if it's going to modify behavior. 

The `if err := ...; err != nil` construction is common and idiomatic, and we don't have max line length constraints in the control plane, so I really don't understand why this change was made (to your own code, according to git blame).

I understand that the current state of our CI system encourages us to maximize the amount of change made for a given PR, but as a general rule I think we should refrain from mixing these sorts of cosmetic-only changes into PRs with other significant changes.

Reply:
Not in the control plane. We don't enforce a line length, nor should we, IMO.